packages/seacas/libraries/ioss/src/Ioss_ElementTopology.h

```diff
@@ -117,14 +122,20 @@ namespace Ioss {
     bool operator!=(const Ioss::ElementTopology &rhs) const;
     bool equal(const Ioss::ElementTopology &rhs) const;
 
+    virtual ElementPermutation *permutation() const;
```

Comment:
Shouldn't be virtual, but you noticed this.  Just noting so we don't forget.

Reply:
Yeah, I'll fix that up